packages/block/src/header.ts

```diff
@@ -201,19 +208,29 @@ export class BlockHeader {
         throw new Error('A base fee for a block can only be set with EIP1559 being activated')
       }
     }
-
     if (this._common.isActivatedEIP(4895)) {
       if (withdrawalsRoot === undefined) {
         throw new Error('invalid header. withdrawalsRoot should be provided')
       }
     } else {
       if (withdrawalsRoot !== undefined) {
         throw new Error(
-          'A withdrawalsRoot for a header can only be provied with EIP4895 being activated'
+          'A withdrawalsRoot for a header can only be provided with EIP4895 being activated'
         )
       }
     }
 
+    if (this._common.isActivatedEIP(4844)) {
```

Comment:
aren't the first segment (true condition ) of `this._common.isActivatedEIP` for 4844 and 4895 redundant since we set our defaults to have some value if these are activated (e.g. see L187)

Reply:
Good point.  Will remove.

Reply:
Good point.  Will remove.